doc-surrealdb_versioned_docs/version-1.x/tutorials/define-a-schema.mdx

```diff
@@ -0,0 +1,230 @@
+
+```surql
+-- Using the CREATE statement to populate the table 
+CREATE user CONTENT {
+    firstName: 'John',
+    lastName: 'Doe',
+    email: 'JohnDoe.com',
+};
+```
+
+The above will return an error because the field must conform to `string::is::email($value)` 
+
+### Inserting fields that don’t exist in the schema
+
+In a schemafull table, since the fields need to be defined before you can populate them. If you add a field that doesn’t exist your data will be ignored. For example, in the user table you have only defined the `firstname` , `lastname` and `email` fields. If you introduce a `photoURI` field without defining the field in the `user` table it will return an error. 
+
+```surql
+-- 2: Add a user with all required fields and an undefined one, 'photoURI'.
+CREATE user CONTENT {
+    firstName: 'John',
+    lastName: 'Doe',
+    email: 'JohnDoe@someemail.com',
+    photoURI: 'photo/yxCFi22Jw2.webp'
+};
+```
+
+## Schemaless (Unstructured) Databases
+
+A schemaless database does not require predefined structures, allowing for more flexible and dynamic data storage. This approach is ideal for applications with evolving data models or when dealing with diverse and unpredictable data formats.
+
+In SurrealDB there are two ways you can define a schemaless table. You can either use any of the data definition statements such as [`CREATE`](/docs/surrealdb/surrealql/statements/create) or [`UPDATE`](/docs/surrealdb/surrealql/statements/update)  and that will make a table based on the [record ID](/docs/surrealdb/surrealql/datamodel/ids#object-based-record-ids) specified. For example:
+
+```surql
+-- Using the CREATE statement 
+CREATE IC_directory:{username: 'johndoe',full_name: 'John Doe'} CONTENT {
+	username: 'johndoe',
+	full_name: 'John Doe',
+	email: 'johndoe@example.com',
+	date_of_birth: "1990-01-01",
+	join_date: "2024-05-30",
+	department: 'Engineering',
+	role: 'Software Engineer',
+	skills: ['Python', 'JavaScript', 'surql'],
+	manager: manager_directory:janesmith,
+	tags: ['full-time', 'remote']
+};
+
+-- Using the UPDATE statement 
+UPDATE manager_directory:janesmith CONTENT {
+	username: 'janesmith',
+	full_name: 'Jane Smith',
+	email: 'janesmith@example.com',
+	date_of_birth: "1985-01-01",
+	join_date: "2019-05-30",
+	department: 'Engineering',
+    role: 'Software Engineer Manager',
+	skills: ['Python', 'JavaScript', 'surql'],
+	report: IC_directory:{ full_name: 'John Doe', username: 'johndoe' },
+	tags: ['full-time', 'remote']
+};
+```
+
+In the example above, you used the `CREATE` statement to make an `IC_directory` table and created an Object ID for this table. ID can be anything you want and is randomly generated if not defined. You have also used the UPDATE statement to make a `manager_directory`, which has `janesmith` as the ID. Learn more about [Record IDs in the documentation](/docs/surrealdb/surrealql/datamodel/ids#object-based-record-ids).
```

Comment:
I think here we could rewrite the "ID can be anything you want..." sentence to this:

Record IDs can be specified by you [in a large number of formats](https://surrealdb.com/docs/surrealdb/surrealql/datamodel/ids), defaulting to a random UUID if you don't specify your own format for the ID.

And then could remove the final "Learn more about..." sentence since the link is now in the previous one.